assets/js/app.js

```diff
@@ -12,8 +11,23 @@ function render(component, domNode) {
     <Provider store={store} children={component} />,
     domNode
   )
-}
 
+  const hamburgerButton = document.getElementById('hamburger-button')
+  const hamburgerMenu = document.getElementById('hamburger-menu')
+  const adminDropdownButton = document.getElementById('admin-dropdown-button')
+  const adminDropdownMenu = document.getElementById('admin-dropdown-menu')
+
+  hamburgerButton.addEventListener('click', () => {
+    hamburgerButton.classList.toggle('collapsed')
+    hamburgerMenu.classList.toggle('navbar__menu--visible')
+  })
+  
+  adminDropdownButton.addEventListener('click', () => {
+    adminDropdownButton.classList.toggle('collapsed')
+    adminDropdownMenu.classList.toggle('navbar__menu-dropdown--visible')
+  })
+}
```

Comment:
@justynawojciechowska 
Current react setup is bad for having this menu as react component, because we don't have SSR. Only table with books is rendered via react and not rendered on phoenix side.

This menu should NOT be in render function, instead it should be initialized on DOM ready.